config/app-enterprise.yaml

```diff
@@ -20,3 +20,11 @@ ZAPPR_DEFAULT_CONFIG:
     veto:
       # veto/blocking a PR = comment that matches this regex
       pattern: "^(:\\-1:|👎)$"
+  specification:
+    title:
+      length: 8
+    body:
+      length: 8
+      verify:
+        contains-url: true
```

Comment:
So the time has come to decide on hyphens vs underscores in longer configuration keys. I think hyphens are alright.